### PR TITLE
Remove apt-cyg installation and check for jq.

### PR DESCRIPTION
**Pull request by steelhead31**
Solves part of #3711 

Remove installation of apt-cyg and jq, as these are installed via the playbooks.

The script now checks if jq is available, and errors if not present and in the path.

### CODE REVIEW

**Comment by sophia-guo** [comment]
cygwin and jq are being available on adoptium jenkins. I think it’s still good to check and install if they aren't exist for local run.

**Review by sophia-guo — APPROVED** [review]
LGTM

**Review by karianna — APPROVED** [review]

**Comment by steelhead31** [comment]
Local tests work ok, so once code freeze is over, this is good to merge.

File: tooling/reproducible/windows_repro_build_compare.sh [hunk 1]
```diff
@@ -138,37 +139,43 @@ Check_Parameters() {
...
-      chmod +x apt-cyg
-      mv apt-cyg /usr/local/bin
-    else
-      echo "Checksum Is Not OK - Exiting"
-      exit 1
-    fi
+    echo "JQ Is Already Installed"
   fi
 
   # Install JQ Where Not Already Installed
```
Comment (sophia-guo) [inline tooling/reproducible/windows_repro_build_compare.sh:177]: This line can be removed.

Comment (steelhead31) [reply tooling/reproducible/windows_repro_build_compare.sh:175]: Done!, just testing...

**Pull request merged**